tests/chainer_tests/training_tests/triggers_tests/test_minmax_trigger.py

```diff
@@ -26,6 +26,10 @@ def epoch(self):
         return 1
 
     @property
+    def epoch_detail(self):
```

Comment:
I am wondering why we modify the unit test of `MinMaxTrigger` as this PR is about `IntervalTrigger`?